package.json

```diff
@@ -1,81 +1,85 @@
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "files": [
-    "*.d.ts",
-    "*.js",
-    "*.map.js",
-    "**/*.d.ts",
-    "**/*.js",
-    "**/*.map.js",
-    "documentation/**/*"
-  ],
-  "scripts": {
-    "build": "yarn clean && yarn build:ts && yarn webpack:prod && yarn webpack:dev",
-    "build:dev": "yarn clean && yarn build:ts-dev && yarn webpack:dev",
-    "build:ts": "tsc -p tsconfig.prod.json",
-    "build:ts-dev": "tsc -p tsconfig.dev.json",
-    "webpack:dev": "webpack --env=dev",
-    "webpack:prod": "webpack --env=prod",
-    "clean": "rm -rf dist",
-    "build-docs": "compodoc -p tsconfig.dev.json -d ./docs",
-    "serve-docs": "compodoc -p tsconfig.dev.json -d ./docs -r 4214 -s",
-    "test": "yarn run build:ts-dev && jasmine './dist/**/*.spec.js'",
-    "test:cover": "nyc -e .ts -x '**/*.spec.ts' jasmine-ts --config=jasmine.json && nyc report -r lcov",
-    "ts-tidy": "npx eslint -c .ts-eslint-tidy-rc.json 'src/**/*.ts' --fix",
-    "prepublish": "npm run build"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/CpanelInc/cpanel-node-api"
-  },
-  "author": "cPanel L.L.C.",
-  "contributors": [
-    {
-      "name": "Aneece Yazdani",
-      "email": "aneece@cpanel.net"
+    "name": "@cpanel/api",
+    "version": "4.0.0-alpha.1",
+    "description": "cPanel API JavaScript and TypeScript interface libraries. This library provides a set of classes for calling cPanel WHM API 1 and UAPI calls. The classes hide much of the complexity of these APIs behind classes the abstract the underlying variances between these API systems. Users of this library can focus on what they want to accomplish rather the having to learn the various complexities of the underlying wire formats for each of the cPanel Products APIs",
+    "main": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "files": [
+        "*.d.ts",
+        "*.js",
+        "*.map.js",
+        "**/*.d.ts",
+        "**/*.js",
+        "**/*.map.js",
+        "documentation/**/*"
+    ],
+    "engines": {
+        "yarn": "YARN NO LONGER USED - use npm instead."
     },
-    {
-      "name": "Sruthi Sanigarapu",
-      "email": "sruthi@cpanel.net"
+    "scripts": {
+        "build:prod": "npm run clean && npm run build:ts && npm run webpack:prod",
+        "build:dev": "npm run clean && npm run build:ts-dev && npm run webpack:dev",
+        "build:ts": "tsc -p tsconfig.prod.json",
+        "build:ts-dev": "tsc -p tsconfig.dev.json",
+        "webpack:dev": "webpack --env dev",
+        "webpack:prod": "webpack --env prod",
+        "clean": "rm -rf dist",
+        "build-docs": "compodoc -p tsconfig.dev.json -d ./docs",
+        "serve-docs": "compodoc -p tsconfig.dev.json -d ./docs -r 4214 -s",
+        "test": "npm run build:ts && jasmine --config=jasmine.json --reporter=jasmine-console-reporter",
```

Comment:
Does running tests against a prod build make it harder to debug while developing? Or do you think it doesn't make much difference because it's a compiled product either way?

Reply:
I'm not sure, but I updated it to use the dev build instead, in case that does make it easier to debug.